Yoast/YoastSEO.js
# Remove taxonomy text length assessment

## Summary

This PR can be summarized in the following changelog entry: Removes the taxonomy text length assessment. This assessment is replaced by the regular text length assessment.

## Relevant technical choices:

* The text length assessment takes over the functionality of the taxonomy text length assessment; the relevant boundaries are passed as arguments. See https://github.com/Yoast/wordpress-seo/pull/9001

## Test instructions

This PR can be tested by following these steps:

* For WordPress SEO check out the branch in https://github.com/Yoast/wordpress-seo/pull/9001. If that branch has already been merged into `feature/recalibration`, check out that branch.
 * For yoastseo.js check out this branch. Use `yarn link` to link it to the correct branch in WordPress SEO.
* Make sure the taxonomy test length assessment still works using these instructions:
  * Add a new category in `Posts > Categories`. Add a description.
  * Vary the length of the description. Make sure there are appropriate feedback messages according to the new boundaries as specified in https://github.com/Yoast/wordpress-seo/pull/9001.

Fixes #1300
